man/kernel-command-line.xml

```diff
@@ -663,12 +663,24 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>hostname=</varname></term>
+
+        <listitem><para>Accepts a hostname to set during early boot. It is used as a default until another
+        hostname is set via e.g. <filename>/etc/hostname</filename> or DHCP.
+        This parameter is intepreted by the kernel to set the initial hostname and by systemd as a fallback
+        if not other hostname is set.</para>
+
+        <xi:include href="version-info.xml" xpointer="v256"/>
+        </listitem>
```

Comment:
I've rebased the Pull Request (sorry for taking so long to do so - I assumed the rebase would include code changes that would require some time to resolve, but it ended up only affecting the documentation).

There is now a CI check that ensures all man page entries have a version set in which they were added.
I've just used the current version + 1 here. Is this the correct way to do it? What happens if the PR does not make it into the next release?